### DISCUSSION TITLE
How to back up attachments

### DISCUSSION THREAD

**Discussion by maple185** [dani-garcia/vaultwarden#1285] [Q&A]
Your environment:WINDOWS10 20h2
Bitwarden_rs version:2.16.1
How the server was installed: Docker image
Clients used: 6.17

https://user-images.githubusercontent.com/54958410/103245026-be151580-4956-11eb-8db2-70c2750ce988.mp4

**Comment by darrenjl0** [comment]
Not the most elegant solution, but I just backup and restore the attachments folder that is mounted into the containers /data folder.
You could also do something like this: https://github.com/ckabalan/bitwarden-attachment-exporter

**Comment by BlackDex** [comment]
If you just want a backup to be able to be restored still using bitwarden_rs then copying those files including the db.sqlite3  file is probably all you need.

Else you can also try https://github.com/vwxyzjn/portwarden .
